OpenRA.Mods.Common/Traits/Render/WithSpriteBody.cs

```diff
@@ -81,8 +81,8 @@ public void PlayCustomAnimation(Actor self, string name, Action after = null)
 
 		public void PlayCustomAnimationRepeating(Actor self, string name)
 		{
-			DefaultAnimation.PlayThen(name,
-				() => PlayCustomAnimationRepeating(self, name));
+			DefaultAnimation.PlayThen(NormalizeSequence(self, name),
```

Comment:
Harvesting animations now appear to be double-normalized.  You should be able to adjust WithHarvestAnimation to account for these upstream checks (which I think are a good idea).